### PR TITLE
Various: New label properties for admin1 records

### LINKED ISSUE #1534: Adjust and add label:* properties to admin1 and admin2
Ensure that all admin1 and admin2 have appropriate `label:eng_*` properties. With any feature that has existing label properties, make sure they are correct, ie: https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data/master/data/856/768/13/85676813.geojson

See also: https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/label

### PR DESCRIPTION
**Pull request by stepps00**
Fixes the admin1 portion of #1534.

This PR includes:
- New `label:eng_x_preferred_longname` properties to admin1 records
- Additional, new `label:eng_us_x_preferred_longname` properties to admin1 records in the US
- New `label:eng_x_preferred_placetype` properties to some admin1
- Cleans up a few issues in Brazil (capitalized names)
- Sanitized some major city admin1 to verify label construction logic worked (Washington DC, etc)

Note: Not all admin1 are included in this PR. Some records do not have properties we can use to construct new label values. I'd like to give this a round of QA before crawling the leftovers.

To do:
  - [x] Add label properties to any record without label properties (or confirm the record is okay without them)
  - [x] Validate record 
  - [x] Remove values from GB
  - [x] Check all new values for duplicate placetypes (like Moscow Oblast Oblast)
  - [ ] Merge

### CODE REVIEW

File: data/110/878/476/1/1108784761.geojson [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
     "edtf:inception":"uuuu",
     "geom:area":0.012831,
-    "geom:area_square_m":152689012.601176,
+    "geom:area_square_m":152689012.60118,
     "geom:bbox":"34.9002538605,-15.8260976184,35.0289508287,-15.698610623",
     "geom:latitude":-15.761288,
     "geom:longitude":34.96266,
     "iso:country":"MW",
+    "label:eng_x_preferred_longname":[
+        "Blantyre City District"
```
Comment (nvkelso) [inline data/110/878/476/1/1108784761.geojson]: Is the `District` necessary here? I'd say no if it's coterminous with the locality.

File: data/115/939/724/3/1159397243.geojson [hunk 2]
```diff
@@ -10,6 +10,9 @@
     "geom:latitude":14.617045,
     "geom:longitude":121.029377,
     "iso:country":"PH",
+    "label:eng_x_preferred_longname":[
+        "National Capital District Region"
```
Comment (nvkelso) [inline data/115/939/724/3/1159397243.geojson]: `"National Capital District"` alone (drop Region)?

File: data/117/561/295/5/1175612955.geojson [hunk 3]
```diff
@@ -5,11 +5,14 @@
...
     "edtf:inception":"uuuu",
     "geom:area":0.00228,
-    "geom:area_square_m":22686182.450553,
+    "geom:area_square_m":22815191.504486,
     "geom:bbox":"14.3196785,35.9306305,14.4074355,35.9983875",
     "geom:latitude":35.962283,
     "geom:longitude":14.355762,
     "iso:country":"MT",
+    "label:eng_x_preferred_longname":[
+        "Mellieha Il-Mellieha"
```
Comment (nvkelso) [inline data/117/561/295/5/1175612955.geojson]: This seems bogus.

Comment (stepps00) [reply data/117/561/295/5/1175612955.geojson]: All regions in Malta were fixed.

File: data/115/939/724/3/1159397243.geojson (hunk 2 above)
Comment (stepps00) [reply data/115/939/724/3/1159397243.geojson]: Agree

File: data/136/069/861/3/1360698613.geojson [hunk 4]
```diff
@@ -10,6 +10,12 @@
     "geom:latitude":52.223728,
     "geom:longitude":-2.16921,
     "iso:country":"GB",
+    "label:eng_x_preferred_longname":[
+        "Worcestershire Administrative County"
```
Comment (nvkelso) [inline data/136/069/861/3/1360698613.geojson]: These are a mouthful...

File: data/110/878/476/1/1108784761.geojson (hunk 1 above)
Comment (stepps00) [reply data/110/878/476/1/1108784761.geojson]: Removed in an upcoming commit..

File: data/136/069/861/3/1360698613.geojson (hunk 4 above)
Comment (stepps00) [reply data/136/069/861/3/1360698613.geojson]: About half of the admin1 in Great Britain have longname labels, which are "* Administrative County". I'd be fine dropping the `label` properties in any record in this country.

**Comment by stepps00** [comment]
I've validated all changes and added a new commit to address comments. 

I was able to add `label:*` properties to all but 76 region records in WOF, list [here](https://github.com/whosonfirst-data/whosonfirst-data/files/3010262/leftovers-that-do-not-get-label.txt). This PR is ready to merge unless there are more values that still look off.

File: data/136/069/861/3/1360698613.geojson (hunk 4 above)
Comment (nvkelso) [reply data/136/069/861/3/1360698613.geojson]: Yes, let's drop them in UK for now.

**Comment by nvkelso** [comment]
Also, regions in California look good in https://github.com/whosonfirst-data/whosonfirst-data/blob/stepps00/labels-1/data/856/886/37/85688637.geojson.

**Comment by stepps00** [comment]
Fix "Moscow Oblast Oblast" and check other cases where we may be duplicating the placetype in the new label value.

**Comment by stepps00** [comment]
Validated all records, took care of all comments. Merging.

**Pull request merged**